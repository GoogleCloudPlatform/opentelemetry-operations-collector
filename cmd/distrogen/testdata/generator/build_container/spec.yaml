# Copyright 2025 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

name: basic-distro

# What this test case is testing.
build_container: ubuntu
collector_cgo: true

display_name: Basic OTel
version: 0.121.0
description: "A basic distribution of the OpenTelemetry Collector"
blurb: "A basic collector distro"
opentelemetry_version: 0.121.0
opentelemetry_stable_version: 1.27.0
binary_name: otelcol-basic
go_version: 1.24.0

feature_gates:
  - exporter.googlemanagedprometheus.intToDouble

docker_repo: us-docker.pkg.dev/pretend-repo/otelcol-basic

components:
  receivers:
    - otlp

  processors:
    - batch
    - memorylimiter

  exporters:
    - otlp

  extensions:
    - pprof

  connectors:
    - forward

  providers:
    - yaml
