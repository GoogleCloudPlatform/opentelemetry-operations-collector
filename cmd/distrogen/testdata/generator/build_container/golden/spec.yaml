name: basic-distro
module: ""
display_name: Basic OTel
description: A basic distribution of the OpenTelemetry Collector
blurb: A basic collector distro
build_container: ubuntu
version: 0.121.0
opentelemetry_version: 0.121.0
opentelemetry_contrib_version: 0.121.0
opentelemetry_stable_version: 1.27.0
go_version: 1.24.0
binary_name: otelcol-basic
build_tags: ""
collector_cgo: true
boringcrypto: false
docker_repo: us-docker.pkg.dev/pretend-repo/otelcol-basic
components:
    receivers:
        - otlp
    processors:
        - batch
        - memorylimiter
    exporters:
        - otlp
    connectors:
        - forward
    extensions:
        - pprof
    providers:
        - yaml
feature_gates:
    - exporter.googlemanagedprometheus.intToDouble
